logging.js

```diff
@@ -235,11 +238,23 @@ export class LoggingClient {
 	}
 
 	_throttle(log) {
+
+		const now = Date.now();
```

Comment:
I've read this has slightly better performance than what this was doing previously with `new Date().getTime()`.